### PR TITLE
Update sequelize

### PR DESCRIPTION
**Pull request by gavrielrh**
Updates to following package(s) (fixing vulnerabilities):
`sequelize` 3.19.0 -> **6.0.0**

There are breaking changes between `v3` and `v4`: https://sequelize.org/v4/manual/tutorial/upgrade-to-v4.html

- Sequelize models are now `ES6` classes, so `instanceMethods` have been moved out of options. They are now set via `prototype`
- `options.order` now only allows arrays / sequelize methods, not string values
- `afterUpdate` is now within the `hooks` options, and not a direct `prototype` of a model

And breaking changes between `v4` and `v5`: https://sequelize.org/master/manual/upgrade-to-v5.html

- Operator aliases are no longer allowed by default (`$lt` must now be `[Op.lt]`)
- `findById` is now `findByPk`

### CODE REVIEW

File: events/membership.js [hunk 1]
```diff
@@ -10,70 +10,75 @@ import User from '../models/user';
 // 'user' and 'secretary' are User objects
 // 'user' = person the email should go to
 // 'secretary' = current SSE secretary
-const sendCongratsEmail = (user, secretary = null) => mailer.sendMail({
```
Comment (billdybas) [inline events/membership.js]: I'm curious why all the formatting changed in these files. Do you know why?

File: package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
     "nodemailer-mailgun-transport": "^1.4.0",
     "pg": "^4.4.1",
     "pg-hstore": "^2.3.2",
-    "sequelize": "^3.19.0",
+    "sequelize": "^6.0.0",
```
Comment (billdybas) [inline package.json]: I see `5.16.0` as the last release: https://github.com/sequelize/sequelize/releases
Should we use that version until there are release notes about 6?

Comment (gavrielrh) [reply package.json]: Definitely. The security advisory suggested @6.0.0, but from what I saw the issue is fixed from like 5.3.0 on, so I'll set this to the latest release.

File: events/membership.js (hunk 1 above)
Comment (gavrielrh) [reply events/membership.js]: That's totally my fault; I haven't used vscode for a hot minute and forgot I had Prettier enabled with format on save (and that the vscode prettier plugin adds its own default prettier settings). Prettier went a... little overzealous.

**Pull request closed without merging**